.github/workflows/indy_image_build.yaml

```diff
@@ -0,0 +1,132 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Build indy os docker images triggered by :
+#  - push on develop branch for latest tags 
+#  - create tag vx.y.z for release tags
+#  - Can be triggered manually
+
+name: Build indy os docker images
+
+on:
+  workflow_dispatch:
+  create:
+    tags:
+      - v*
+  push:
+    branches:
+      - develop
+    paths:
+      - 'platforms/hyperledger-indy/images/**'   
+      - '.github/workflows/indy_images_build.yml'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PATH: ghcr.io/${{ github.repository_owner }}
+
+jobs:
+  # on push to develop branch, create docker tags latest
+  indyOS-latest:
+    if: ${{ startsWith(github.ref, 'refs/heads/develop') }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Login to the container registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+       
+    - name: Build and push indyOS indy-cli latest  
+      id: indy-cli_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: platforms/hyperledger-indy/images/indy-cli
+        push: true
+        tags: ${{ env.IMAGE_PATH }}/indy-cli:latest            
```

Comment:
this image should be called bevel-indy-ledger-txn:latest

Reply:
changed